modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache

```diff
@@ -369,3 +369,40 @@ export class {{classname}} extends BaseAPI {
     {{/operation}}
 }
 {{/operations}}
+{{#hasEnums}}
```

Comment:
**nitpick:** this seems redundant - since you render output for each collection element matching the filter (`{{#allParams}} {{#isEnum}}`). If the isEnum condition is never satisfied, nothing is rendered, even without the top-level wrapping hasEnum check

Reply:
removed these.